[2016-12-18 18:32:19] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2016-12-18 18:32:41] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeFileSessionHandler.php\",\"line\":51})"} []
[2016-12-18 18:32:41] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeFileSessionHandler.php line 52 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeFileSessionHandler.php:52)"} []
[2016-12-18 18:32:41] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php\",\"line\":331})"} []
[2016-12-18 18:32:41] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php\",\"line\":331})"} []
[2016-12-18 18:32:42] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:32:42] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:32:42] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:32:42] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:13] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:33:13] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:33:13] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:13] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:14] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:33:14] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:33:14] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:14] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:20] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:33:20] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:33:20] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:20] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:21] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:33:21] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:33:21] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:21] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:24] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:33:24] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:33:24] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:24] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:26] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:33:26] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:33:26] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:33:26] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:34:11] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2016-12-18 18:34:16] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:34:16] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:34:16] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:34:16] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:34:17] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:34:17] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:34:17] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:34:17] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:50:00] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2016-12-18 18:50:12] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:50:12] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:50:12] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:50:12] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:50:13] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-18 18:50:14] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-18 18:50:14] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-18 18:50:14] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:47:55] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2016-12-26 13:48:13] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-26 13:48:13] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-26 13:48:13] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:48:13] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:48:15] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-26 13:48:15] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-26 13:48:15] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:48:15] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:48:19] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-26 13:48:19] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-26 13:48:19] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:48:19] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:48:20] php.DEBUG: Warning: mkdir(): Permission denied {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":291})"} []
[2016-12-26 13:48:20] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 292 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/home/hugo/Documents/Tests_Symfony/test_nlpf/app/../var/sessions/dev\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:292)"} []
[2016-12-26 13:48:20] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:48:20] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:30] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2016-12-26 13:50:35] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:35] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:35] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_controller":"web_profiler.controller.profiler:panelAction","token":"e005a6","_route":"_profiler"},"request_uri":"http://localhost:8000/_profiler/e005a6","method":"GET"} []
[2016-12-26 13:50:36] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_controller":"web_profiler.controller.profiler:searchBarAction","_route":"_profiler_search_bar"},"request_uri":"http://localhost:8000/_profiler/search_bar","method":"GET"} []
[2016-12-26 13:50:46] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_controller":"web_profiler.controller.profiler:panelAction","token":"e005a6","_route":"_profiler"},"request_uri":"http://localhost:8000/_profiler/e005a6?panel=exception","method":"GET"} []
[2016-12-26 13:50:46] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:46] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:46] request.INFO: Matched route "_profiler_exception_css". {"route":"_profiler_exception_css","route_parameters":{"_controller":"web_profiler.controller.exception:cssAction","token":"e005a6","_route":"_profiler_exception_css"},"request_uri":"http://localhost:8000/_profiler/e005a6/exception.css","method":"GET"} []
[2016-12-26 13:50:46] request.INFO: Matched route "_profiler_exception". {"route":"_profiler_exception","route_parameters":{"_controller":"web_profiler.controller.exception:showAction","token":"e005a6","_route":"_profiler_exception"},"request_uri":"http://localhost:8000/_profiler/e005a6/exception","method":"GET"} []
[2016-12-26 13:50:47] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_controller":"web_profiler.controller.profiler:searchBarAction","_route":"_profiler_search_bar"},"request_uri":"http://localhost:8000/_profiler/search_bar?panel=exception","method":"GET"} []
[2016-12-26 13:50:58] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2016-12-26 13:50:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-12-26 13:50:58] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:58] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:59] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"b7fb7a","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/b7fb7a","method":"GET"} []
[2016-12-26 13:50:59] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2016-12-26 13:50:59] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2017-01-06 09:50:20] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/MAMP/htdocs/nlpf_symfony/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2017-01-11 13:52:06] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2017-01-11 13:52:18] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 13:52:18] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 13:52:18] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-01-11 13:52:18] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-11 13:52:19] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: could not find driver" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php line 85 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: could not find driver at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php:85, Doctrine\\DBAL\\Driver\\PDOException(code: 0): could not find driver at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 0): could not find driver at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2017-01-11 13:52:20] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"ebfa59","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/ebfa59","method":"GET"} []
[2017-01-11 13:52:20] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 13:52:20] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 13:56:48] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2017-01-11 13:56:53] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-01-11 13:56:53] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-11 13:56:53] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 13:56:53] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 13:56:53] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
[2017-01-11 13:56:54] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':  SQLSTATE[HY000]: General error: 1 no such table: project" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php line 58 {"exception":"[object] (Doctrine\\DBAL\\Exception\\TableNotFoundException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':\n\nSQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php:58, Doctrine\\DBAL\\Driver\\PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:106, PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:104)"} []
[2017-01-11 13:56:55] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"df8d21","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/df8d21","method":"GET"} []
[2017-01-11 13:56:55] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 13:56:55] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:46] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2017-01-11 14:06:50] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-01-11 14:06:50] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-11 14:06:50] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:50] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:50] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
[2017-01-11 14:06:50] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':  SQLSTATE[HY000]: General error: 1 no such table: project" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php line 58 {"exception":"[object] (Doctrine\\DBAL\\Exception\\TableNotFoundException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':\n\nSQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php:58, Doctrine\\DBAL\\Driver\\PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:106, PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:104)"} []
[2017-01-11 14:06:51] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"2b865a","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/2b865a","method":"GET"} []
[2017-01-11 14:06:51] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:51] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:56] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-01-11 14:06:56] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-11 14:06:56] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:56] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:56] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
[2017-01-11 14:06:56] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':  SQLSTATE[HY000]: General error: 1 no such table: project" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php line 58 {"exception":"[object] (Doctrine\\DBAL\\Exception\\TableNotFoundException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':\n\nSQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php:58, Doctrine\\DBAL\\Driver\\PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:106, PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:104)"} []
[2017-01-11 14:06:57] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"49ae79","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/49ae79","method":"GET"} []
[2017-01-11 14:06:57] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:06:57] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:05] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-01-11 14:07:05] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-11 14:07:05] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:05] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:05] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
[2017-01-11 14:07:05] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':  SQLSTATE[HY000]: General error: 1 no such table: project" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php line 58 {"exception":"[object] (Doctrine\\DBAL\\Exception\\TableNotFoundException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':\n\nSQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php:58, Doctrine\\DBAL\\Driver\\PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:106, PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:104)"} []
[2017-01-11 14:07:06] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"292a20","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/292a20","method":"GET"} []
[2017-01-11 14:07:06] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:06] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:46] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2017-01-11 14:07:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-11 14:07:46] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:46] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:46] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
[2017-01-11 14:07:46] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\TableNotFoundException: "An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':  SQLSTATE[HY000]: General error: 1 no such table: project" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php line 58 {"exception":"[object] (Doctrine\\DBAL\\Exception\\TableNotFoundException(code: 0): An exception occurred while executing 'SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0':\n\nSQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php:58, Doctrine\\DBAL\\Driver\\PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:106, PDOException(code: HY000): SQLSTATE[HY000]: General error: 1 no such table: project at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:104)"} []
[2017-01-11 14:07:47] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"c63dc4","_route":"_wdt"},"request_uri":"http://127.0.0.1:8000/_wdt/c63dc4","method":"GET"} []
[2017-01-11 14:07:47] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:07:47] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:29:24] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php\",\"line\":331})"} []
[2017-01-11 14:29:24] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php\",\"line\":331})"} []
[2017-01-11 14:29:24] doctrine.DEBUG: SELECT name FROM sqlite_master WHERE type = 'table' AND name != 'sqlite_sequence' AND name != 'geometry_columns' AND name != 'spatial_ref_sys' UNION ALL SELECT name FROM sqlite_temp_master WHERE type = 'table' ORDER BY name [] []
[2017-01-11 14:29:24] doctrine.DEBUG: SELECT name FROM sqlite_master WHERE type = 'table' AND name != 'sqlite_sequence' AND name != 'geometry_columns' AND name != 'spatial_ref_sys' UNION ALL SELECT name FROM sqlite_temp_master WHERE type = 'table' ORDER BY name [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE TABLE users (id INTEGER NOT NULL, firstname VARCHAR(255) NOT NULL, lastname VARCHAR(255) NOT NULL, password VARCHAR(255) NOT NULL, email VARCHAR(255) NOT NULL, PRIMARY KEY(id)) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE UNIQUE INDEX UNIQ_1483A5E9E7927C74 ON users (email) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE TABLE project (id INTEGER NOT NULL, owner_id INTEGER DEFAULT NULL, name VARCHAR(255) NOT NULL, description VARCHAR(255) NOT NULL, current INTEGER NOT NULL, created DATETIME NOT NULL, PRIMARY KEY(id)) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE INDEX IDX_2FB3D0EE7E3C61F9 ON project (owner_id) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE TABLE ConterpartsProject (Project_idProject INTEGER NOT NULL, Conterpart_idConterpart INTEGER NOT NULL, PRIMARY KEY(Project_idProject, Conterpart_idConterpart)) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE INDEX IDX_701C45F395CC31 ON ConterpartsProject (Project_idProject) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE INDEX IDX_701C45F37A4E7322 ON ConterpartsProject (Conterpart_idConterpart) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE TABLE invest (id INTEGER NOT NULL, investor_id INTEGER DEFAULT NULL, project_id INTEGER DEFAULT NULL, conterpart_id INTEGER DEFAULT NULL, PRIMARY KEY(id)) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE INDEX IDX_EB095F869AE528DA ON invest (investor_id) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE INDEX IDX_EB095F86166D1F9C ON invest (project_id) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE INDEX IDX_EB095F8632855D23 ON invest (conterpart_id) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE TABLE conterpart (id INTEGER NOT NULL, name VARCHAR(255) NOT NULL, description VARCHAR(255) NOT NULL, value INTEGER NOT NULL, PRIMARY KEY(id)) [] []
[2017-01-11 14:29:24] doctrine.DEBUG: CREATE UNIQUE INDEX UNIQ_34A65D5D5E237E06 ON conterpart (name) [] []
[2017-01-11 14:31:23] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-01-11 14:31:23] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-11 14:31:23] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:23] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:23] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
[2017-01-11 14:31:23] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [1] []
[2017-01-11 14:31:23] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [3] []
[2017-01-11 14:31:23] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [2] []
[2017-01-11 14:31:23] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [4] []
[2017-01-11 14:31:23] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/bootstrap.min.css" (from "http://localhost:8000/")" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 7220 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /css/bootstrap.min.css\" (from \"http://localhost:8000/\") at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:7220, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-11 14:31:23] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:23] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:25] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap.min.js" (from "http://localhost:8000/")" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 7220 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap.min.js\" (from \"http://localhost:8000/\") at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:7220, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-11 14:31:25] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:25] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:26] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/jquery.js" (from "http://localhost:8000/")" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 7220 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/jquery.js\" (from \"http://localhost:8000/\") at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:7220, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-11 14:31:26] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:26] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/3-col-portfolio.css" (from "http://localhost:8000/")" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 7220 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /css/3-col-portfolio.css\" (from \"http://localhost:8000/\") at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:7220, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-11 14:31:27] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:27] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:29] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/bootstrap.min.css" (from "http://localhost:8000/")" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 7220 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /css/bootstrap.min.css\" (from \"http://localhost:8000/\") at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:7220, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-11 14:31:29] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:29] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_controller":"web_profiler.controller.profiler:toolbarAction","token":"14551a","_route":"_wdt"},"request_uri":"http://localhost:8000/_wdt/14551a","method":"GET"} []
[2017-01-11 14:31:30] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:30] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/jquery.js" (from "http://localhost:8000/")" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 7220 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/jquery.js\" (from \"http://localhost:8000/\") at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:7220, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-11 14:31:30] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:30] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap.min.js" (from "http://localhost:8000/")" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 7220 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap.min.js\" (from \"http://localhost:8000/\") at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:7220, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-11 14:31:32] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-11 14:31:32] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":4517})"} []
[2017-01-12 15:20:02] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2017-01-12 15:21:43] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/ServerCommand.php\",\"line\":59})"} []
[2017-01-12 15:21:45] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php\",\"line\":331})"} []
[2017-01-12 15:21:45] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php\",\"line\":331})"} []
[2017-01-12 15:21:46] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /api/users"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /api/users\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-12 15:21:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /api/user"" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php line 4171 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /api/user\" at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php:4171, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:366)"} []
[2017-01-12 15:21:55] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2017-01-12 15:21:55] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2017-01-12 15:22:01] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2017-01-12 15:22:01] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-12 15:22:01] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2017-01-12 15:22:01] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2017-01-12 15:22:01] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
[2017-01-12 15:22:01] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [1] []
[2017-01-12 15:22:01] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [3] []
[2017-01-12 15:22:01] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [2] []
[2017-01-12 15:22:01] doctrine.DEBUG: SELECT t0.id AS id_1, t0.firstname AS firstname_2, t0.lastname AS lastname_3, t0.password AS password_4, t0.email AS email_5 FROM users t0 WHERE t0.id = ? [4] []
[2017-01-12 15:23:14] request.INFO: Matched route "app_project_getprojects". {"route":"app_project_getprojects","route_parameters":{"_controller":"AppBundle\\Controller\\ProjectController::getProjectsAction","_route":"app_project_getprojects"},"request_uri":"http://127.0.0.1:8000/api/projects","method":"GET"} []
[2017-01-12 15:23:14] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-01-12 15:23:14] php.DEBUG: Warning: include(/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php): failed to open stream: No such file or directory {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2017-01-12 15:23:14] php.DEBUG: Warning: include(): Failed opening '/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/annotations.php' for inclusion (include_path='.:/usr/share/php') {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/home/hugo/Documents/Tests_Symfony/test_nlpf/var/cache/dev/classes.php\",\"line\":2457})"} []
[2017-01-12 15:23:14] doctrine.DEBUG: SELECT t0.id AS id_1, t0.name AS name_2, t0.description AS description_3, t0.current AS current_4, t0.created AS created_5, t0.owner_id AS owner_id_6 FROM project t0 [] []
